pkg/assembler/backends/ent/migrate/migrations/20250122170741_ent_diff.sql

```diff
@@ -0,0 +1,18 @@
+-- Truncate all of the data and remove older certify legal entries
```

Comment:
Truncate the tables such that all the old values get removed and when the certifier runs again, only new values will be added.